### PR TITLE
[medDataSources] compilation error on Ubuntu + accent err

### PR DESCRIPTION
**Pull request by mathildemerle**
Solve a compilation error on Ubuntu 20.04 (in getModelIndex lvalue bug), and fix git diff problems with french accents in comments.

:m:

### CODE REVIEW

File: src/layers/medCore/source/virtualRepresentation/medVirtualRepresentation.cpp [hunk 1]
```diff
@@ -790,7 +790,8 @@ void medVirtualRepresentation::removeTooOldEntry()
 QModelIndex medVirtualRepresentation::getModelIndex(medDataIndex index)
 {
     QModelIndex indexRes;
-    QString indexAsString = indexToFileSysPath(index.asString());
```
Comment (Florent2305) [inline src/layers/medCore/source/virtualRepresentation/medVirtualRepresentation.cpp:793]: Thanks for correction but a better way consist to modify fonctions signatures by
```
QString indexToFileSysPath(const QString &index); 
```
into medDataHub.h

**Review by Florent2305 — CHANGES_REQUESTED** [review]
Thanks for corrections but a little change is required

File: src/layers/medCore/source/virtualRepresentation/medVirtualRepresentation.cpp (hunk 1 above)
Comment (mathildemerle) [reply src/layers/medCore/source/virtualRepresentation/medVirtualRepresentation.cpp]: thx, PR updated with the changes

**Review by Florent2305 — APPROVED** [review]
Ok Thx

**Pull request merged**